client/.eslintrc.js

```diff
@@ -28,5 +28,7 @@ module.exports = {
     'import/no-extraneous-dependencies': 'off',
     '@typescript-eslint/no-unused-vars': 'error',
     '@typescript-eslint/no-explicit-any': 'error',
+    'no-shadow': 'off',
```

Comment:
had issues with defining enums in types files. no-shadow did the trick

https://stackoverflow.com/questions/63961803/eslint-says-all-enums-in-typescript-app-are-already-declared-in-the-upper-scope

Reply:
hmm at the same time, I'm not sure if adding this line 
`// eslint-disable-next-line no-shadow` 
to every enum is the right approach either.

I took a look at the codebase and we already have another instance of this where we silence the error just because we use an enum. I personally try to avoid disabling, but for something as common and fundamental as an enum, I'd rather avoid using a comment to silence.

Reading up on the SO post and `typescript-eslint` docs [here](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-shadow.md), it seems like the base rule is conflicting with the ts rule which is why we define `no-shadow` as none, but redefine the ts layer as ['error'].

With this new context, I'm thinking of keeping this change especially since its what the maintiner suggests aswell [here](https://github.com/typescript-eslint/typescript-eslint/issues/2561)

Reply:
ok so the error was with the original eslint and not typescript-eslint, that's fine then